src/pages/underConstruction/construction.ts

```diff
@@ -15,6 +17,11 @@ export class ConstructionPage {
         this.navCtrl = navCtrl;
         this.pageTitle = navParams.get('linkParam');
     }
+
+    openLink(){
+        this.navCtrl.setRoot(DocumentationPage, { 'linkParam': "https://aerogear.org/community/#guides" });
```

Comment:
also can we move all the links for docs to 1 place so that we can easily update them if we need to?